operator/config/crd/bases/core.cnrm.cloud.google.com_configconnectors.yaml

```diff
@@ -93,6 +93,22 @@ spec:
                 - cluster
                 - namespaced
                 type: string
+              stateIntoSpec:
+                description: StateIntoSpec is the user override of the default value
+                  for the 'cnrm.cloud.google.com/state-into-spec' annotation if the
+                  annotation is unset for a resource. If the field is set in both
+                  the ConfigConnector object and the ConfigConnectorContext object
+                  is in the namespaced mode, then the value in the ConfigConnectorContext
+                  object will be used. 'absent' means that unspecified fields in the
+                  resource spec stay unspecified after successful reconciliation.
+                  'merge' means that unspecified fields in the resource spec are populated
+                  after a successful reconciliation if those unspecified fields are
+                  computed/defaulted by the API. It is only applicable to resources
+                  supporting the 'merge' option.
```

Comment:
nit: should we capitalize `merge`/ `absent` here and for CCC?

Reply:
Good point. Done!